simulator/opendc-experiments/opendc-experiments-allocateam/tools/run.sh

```diff
@@ -14,12 +16,19 @@ command -v realpath > /dev/null 2>&1 || realpath() {
 
 # Check if data directory exists and ask user what to do as we don't want to remove data
 if [ -d "$DATA_DIR" ]; then
```

Comment:
It would be good to create a different data directory for the reproducibility test and that this directory is deleted afterwards, but it's not necessary. It would just be a nice-to-have.